### PR TITLE
GitHub Actions CI: Add MS Edge to matrix

### PR DESCRIPTION
**Pull request by San4es**

### CODE REVIEW

**Review by nightskylark — DISMISSED** [review]

**Review by churkin — DISMISSED** [review]

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -60,8 +62,14 @@ jobs:
...
         $url = "$runner_host/run?notimers=true&$env:JQUERY&constellation=$env:CONSTEL"
-        & "C:\Program Files\Internet Explorer\iexplore.exe" "$url"
-        Write-Host "IE started"
+        $browser = $env:BROWSER
+        $browserPaths = @{
+          IE = "C:\Program Files\Internet Explorer\iexplore.exe";
+          MSEdge = "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe";
+        }
+        Write-Host "Starting $browser with $url"
+        Start-Process -File $browserPaths[$browser] -ArgumentList "$url"
```
Comment (San4es) [inline .github/workflows/CI.yml]: @IgnatovDan suggests:
```suggestion
        Start-Process -File "$browserPaths[$browser]" -ArgumentList "$url"
```

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -60,8 +62,14 @@ jobs:
...
         Write-Host "Runner started"
         $url = "$runner_host/run?notimers=true&$env:JQUERY&constellation=$env:CONSTEL"
-        & "C:\Program Files\Internet Explorer\iexplore.exe" "$url"
-        Write-Host "IE started"
+        $browser = $env:BROWSER
+        $browserPaths = @{
+          IE = "C:\Program Files\Internet Explorer\iexplore.exe";
+          MSEdge = "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe";
+        }
+        Write-Host "Starting $browser with $url"
```
Comment (San4es) [inline .github/workflows/CI.yml]: @IgnatovDan suggests:
```suggestion
        Write-Host "Starting $browserPaths[$browser] with $url"
```

File: .github/workflows/CI.yml (hunk 1 above)
Comment (San4es) [reply .github/workflows/CI.yml]: https://github.com/DevExpress/DevExtreme/runs/1308840611?check_suite_focus=true

File: .github/workflows/CI.yml (hunk 2 above)
Comment (San4es) [reply .github/workflows/CI.yml]: https://github.com/DevExpress/DevExtreme/runs/1309891106?check_suite_focus=true#step:6:45

File: .github/workflows/CI.yml (hunk 1 above)
Comment (San4es) [reply .github/workflows/CI.yml]: Done in https://github.com/DevExpress/DevExtreme/pull/15033/commits/cbc21eca69966f45fb1529015c47fdda93490455.

File: .github/workflows/CI.yml (hunk 2 above)
Comment (San4es) [reply .github/workflows/CI.yml]: Done in cbc21ec.

**Review by IgnatovDan — DISMISSED** [review]

**Review by ksercs — APPROVED** [review]

**Review by churkin — APPROVED** [review]

**Review by vconst — APPROVED** [review]

**Pull request merged**